lib/terraforming/resource/elasti_cache_cluster.rb

```diff
@@ -57,7 +57,7 @@ def tfstate
       private
 
       def cache_clusters
-        @client.describe_cache_clusters(show_cache_node_info: true).cache_clusters
+        @client.describe_cache_clusters(show_cache_node_info: true).map(&:cache_clusters).flatten
```

Comment:
__[rubocop]__
* Line is too long. [97/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)